tests/io/vasp/test_sets.py

```diff
@@ -2094,3 +2094,11 @@ def test_as_from_dict(self):
 
 def test_vasp_input_set_alias():
     assert VaspInputSet is VaspInputGenerator
+
+
+def test_dict_set_alias():
+    assert isinstance(DictSet(), VaspInputSet)
+    with pytest.warns(
+        FutureWarning, match="DictSet is deprecated, and will be removed on 2025-12-31\n; use VaspInputSet"
```

Comment:
@DanielYang59 looks like your recent changes to `@deprecated` in `monty` put the new line in front of the semicolon. looks weird to start a line with a semicolon

```diff
- 2025-12-31\n;
+ 2025-12-31;\n
```

Reply:
Thanks for catching this and apologize, I would push a PR to fix this ASAP.

https://github.com/materialsvirtuallab/monty/blob/7568d6abcfc8dec14ba3022523656d5f9217835e/monty/dev.py#L94-L109

It looks the semicolon was original there, and the deprecation message we insert was too long so a line break was inserted. It would be a bit tricky to properly format this without breaking, I would think about this later
```
        msg = f"{old.__name__} is deprecated"


        if deadline is not None:
            msg += f", and will be removed on {_deadline:%Y-%m-%d}\n"


        if replacement is not None:
            if isinstance(replacement, property):
                r = replacement.fget
            elif isinstance(replacement, (classmethod, staticmethod)):
                r = replacement.__func__
            else:
                r = replacement
            msg += f"; use {r.__name__} in {r.__module__} instead."


        if message:
            msg += "\n" + message
```

Reply:
One other change in monty to make if you can add it to the same PR for the message change: this decorator is just for functions. Would be great if there's a way to decorate classes too